includes/email-templates/header.php

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+  <meta name="format-detection" content="telephone=no">
+  <meta name="viewport" content="initial-scale=1.0">
+
+  <!-- So that mobile webkit will display zoomed in -->
+  <link rel="stylesheet" type="text/css" href="https://s3.amazonaws.com/web.ucf.edu/email/common-assets/stylesheet.css">
+  <title>
+    <?php the_title(); ?>
```

Comment:
Because `the_title()` returns unfiltered values, HTML tags in the post title show up in `<title>` tags--we'll need to filter out HTML from these values.  `wp_strip_all_tags()` could be useful here

Reply:
In one of the Provost emails, they used an <em> tag in the title. I don't think we want to strip out all tags.

Reply:
I'm talking about filtering the email title _just_ in the actual `<title>` tag, not within the email copy itself.  This is what my browser tab looks like in FF, where you can see the literal `<em>` tag:

![screen shot 2019-01-18 at 9 10 08 am](https://user-images.githubusercontent.com/1649216/51391797-11db1b00-1b01-11e9-8fa8-f08e305df597.png)

Reply:
Thanks, I see what you mean. I will add this in the new plugin.